modules/libcom/RTEMS/Makefile

```diff
@@ -27,8 +31,12 @@ rtemsCom_SRCS += setBootConfigFromNVRAM.c
 rtemsCom_SRCS += epicsRtemsInitHookPre.c
 rtemsCom_SRCS += epicsRtemsInitHookPost.c
 rtemsCom_SRCS += epicsMemFs.c
+rtemsCom_SRCS += epicsNtp.c
 
-ifeq ($(RTEMS_BSP),pc386)
+ifeq ($(RTEMS_TARGET_CPU),i386)
+rtemsCom_SRCS += ne2kpci.c
+endif
+ifeq ($(RTEMS_BSP),pc686)
 rtemsCom_SRCS += ne2kpci.c
 endif
```

Comment:
Can this condition include ne2kpci.c twice?

Reply:
> Can this condition include ne2kpci.c twice?

Yeah, looks like it was a merge issue.  I think only checking for CPU=i386 is sufficient, so I removed the BSP=pc686 check.